Source/Core/AudioCommon/WaveFile.h

```diff
@@ -19,14 +19,18 @@
 
 #include "Common/CommonTypes.h"
 #include "Common/File.h"
-#include "Common/NonCopyable.h"
 
-class WaveFileWriter : NonCopyable
+class WaveFileWriter
 {
 public:
   WaveFileWriter();
   ~WaveFileWriter();
 
+  WaveFileWriter(const WaveFileWriter&) = delete;
+  WaveFileWriter& operator=(const WaveFileWriter&) = delete;
+  WaveFileWriter(WaveFileWriter&&) = delete;
```

Comment:
For many of the classes, including this one, the reason was that the destructor wasn't going to work as intended after moving. Though for this class in particular, I'm not completely certain that the destructor won't work as intended, but it does seem like it.

There were a few classes where I wasn't really sure if I should make them noncopyable: `Wiimote` and `NativeVertexFormat` (not sure why these were non-copyable... maybe for preventing slicing derived classes?), and  `Settings` and `WiiSockMan` (they're singletons and only have private constructors, so not being able to move doesn't matter, though I suppose there isn't really any harm in allowing moving).